strings.asm

```diff
@@ -161,21 +164,22 @@ es_returnstack: .text "RS", 0
 ; "envs_".
 
 ; These return a single-cell number
-envs_cs:        .text 15, "/COUNTED-STRING"
-envs_hold:      .text 5, "/HOLD"
-envs_pad:       .text 4, "/PAD"
-envs_aub:       .text 17, "ADDRESS-UNIT-BITS"
-envs_floored:   .text 7, "FLOORED"
-envs_max_char:  .text 8, "MAX-CHAR"
-envs_max_n:     .text 5, "MAX-N"
-envs_max_u:     .text 5, "MAX-U"
-envs_rsc:       .text 18, "RETURN-STACK-CELLS"
-envs_sc:        .text 11, "STACK-CELLS"
-envs_wl:        .text 9, "WORDLISTS"
+envs_cs:        .text "/COUNTED-STRING"
```

Comment:
I noticed ENVIRONMENT? could be smaller/simpler/quicker using the difference in pointers to calculate the lengths rather than encoding explicitly.